package.json

```diff
@@ -50,18 +50,20 @@
     "event-stream": "3.3.2",
     "express": "4.13.4",
     "fs-extra": "0.30.0",
+    "he": "^1.1.0",
```

Comment:
Also, `he` is just an encoder, is it not possible for us to do this ourselves, instead of adding another dependency?